### ISSUE TITLE
[Usability] A11y_Manage.IIS.Net_Websites: Unable to navigate to the menu items of the "More" dropdown using arrow keys.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#106]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106

**Repro Steps:**
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till "Web Sites" in the left menu section and Press Enter.
5. Tab till "More" button and Press Enter.
6. Verify that user is able to navigate to the menu items of the dropdown using arrow keys.

**Actual Result:**
Unable to navigate to the menu items of the "More" dropdown using arrow keys.

**Expected Result:**
User should be able to navigate to the menu items of the "More" dropdown using arrow keys.

**User Impact:**
Navigating through the list items of menu items of the dropdown using arrow key is the standard method used by keyboard only users. This navigation method should work here as well, otherwise user may assume that menu items are not accessible.

**Attachment:**
[Bug 106_Usability issue_unable to navigate inside the dropdown list items using arrow keys.pptx](https://github.com/Microsoft/IIS.WebManager/files/2323715/Bug.106_Usability.issue_unable.to.navigate.inside.the.dropdown.list.items.using.arrow.keys.pptx)

**Comment by jhkimnew** [comment]
Fixed
